docs/resources/custom_image.md

```diff
@@ -1,5 +1,6 @@
 ---
 page_title: "DigitalOcean: digitalocean_custom_image"
+subcategory: "Account"
```

Comment:
```suggestion
subcategory: "Backups & Snapshots"
```

If we're following the pattern from the control panel, custom images are found on a tab on the "Backups & Snapshots" page.